WordPress/src/main/java/org/wordpress/android/ui/engagement/LikedItemViewHolder.kt

```diff
@@ -72,5 +74,12 @@ class LikedItemViewHolder(
                 likedItem.likedItemPostId
             )
         }
+        val isDarkTheme = itemView.resources.configuration.isDarkTheme()
+        val color = ContextCompat.getColor(
+            itemView.context,
+            if (isDarkTheme) R.color.white_translucent_10
```

Comment:
Nit pick suggestion: Wdyt of moving the dark theme color logic in the xml resource files? E.g. set the `header_root_view.cardBackgroundColor` to a new color resource and differentiate with values in `values/colors.xml` and`values-night/colors.xml`

Reply:
This makes sense to me, I've extracted the background color to `colors.xml`. Thank you for the suggestion!